3/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -166,10 +166,20 @@ zookeeper_validate() {
 
     # ZooKeeper server list validations
     if [[ -n $ZOO_SERVERS ]]; then
+        idFlag=false
+        if [[ "$ZOO_SERVERS" == *"::"* ]]; then
+            idFlag=true
+        fi
```

Comment:
Please do not use underscore instead of camelCase to be consistent with the rest of the libraries.
```suggestion
        server_id_with_jumps=no
        [[ "$ZOO_SERVERS" == *"::"* ]] && server_id_with_jumps=yes
```